client/my-sites/site-settings/jetpack-sync-panel/index.jsx

```diff
@@ -0,0 +1,175 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import classNames from 'classnames';
+import get from 'lodash/get';
+import throttle from 'lodash/throttle';
+import reduce from 'lodash/reduce';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+import Notice from 'components/notice';
+import ProgressBar from 'components/progress-bar';
+import { getSyncStatus, scheduleJetpackFullysync } from 'state/jetpack-sync/actions';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { selectSyncStatus, selectFullSyncRequest } from 'state/jetpack-sync/selectors';
+
+/*
+ * Module variables
+ */
+
+const JetpackSyncPanel = React.createClass( {
+	displayName: 'JetpackSyncPanel',
+
+	componentWillMount() {
+		this.throttledPoller = throttle( this.fetchSyncStatus, 4000 );
```

Comment:
Can we leave a comment as to how long this is ? I am guessing 4 second? 
Also might be nice to turn it into a constant.